srcalc/src/main/java/gov/va/med/srcalc/domain/model/NumericalTerm.java

```diff
@@ -1,10 +1,11 @@
 package gov.va.med.srcalc.domain.model;
 
-import gov.va.med.srcalc.domain.variable.NumericalVariable;
+import gov.va.med.srcalc.domain.variable.*;
 
 import java.util.Objects;
 
-import javax.persistence.*;
+import javax.persistence.Embeddable;
+import javax.persistence.ManyToOne;
```

Comment:
My Eclipse is set to use the * import when there are 3 or more classes imported. I figured this diff would make you happy!